Objects/codeobject.c

```diff
@@ -2010,8 +2040,22 @@ _PyCode_Hydrate(PyCodeObject *code)
             return NULL;
         }
     }
-    Py_INCREF(pyc->names);
-    code->co_names = pyc->names;  // The items may still be NULL!!!
+
+    if (code->co_strings_start) {
+        if (code->co_strings_size) {
```

Comment:
This is the only place we use the co_strings_size field. Maybe we can do without?

Reply:
But how would you know how large a tuple to create in the line below?